app/src/main/res/layout/activity_browser.xml

```diff
@@ -134,7 +134,7 @@
             android:id="@+id/autoCompleteSuggestionsList"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="@color/charcoalGrey"
+            android:background="@color/charcoalGray"
```

Comment:
oooh, controversial! _rolls up sleeve for a grammar fight_

I kid... I'm ok with preferring 🇺🇸 usage - especially fitting I suppose since it's juxtaposed with `color`!

I'll keep this in mind when i'm defining colours in future.